3.0-RELEASE-NOTES.md

```diff
@@ -27,3 +27,67 @@ To disable a static remote method
 
 To disable a prototype remote method:
 `disableMethodByName('prototype.updateAttributes')`
+
+## New error-handler for rest-adapter
+
+The REST adapter (typically created via `loopback.rest()`) uses a new error
+handler implementation that provides more secure configuration out of the box.
+
+The environment setting `NODE_ENV='production'` is no longer supported,
+production vs. debug mode is controlled exclusively by configuration.
+Production environment is assumed by default, the insecure debug mode
+must be explicitely turned on.
+
+You can learn more about the rationale behind the new handler in
+[this comment](https://github.com/strongloop/loopback/issues/1650#issuecomment-161920555)
+
+User can specific options in their applications in `config.json` as follow:
```

Comment:
hmm it works in my sand-box
i think the behavior comes from https://github.com/strongloop/strong-remoting/blob/aa391d2e65460196ae2a59493f9dd4844622f35d/lib/rest-adapter.js#L315